pkg/sync/v5/data/ConfigMap/openshift-azure-logging/fluentd-config.yaml

```diff
@@ -9,19 +9,58 @@ data:
       </storage>
       tag logs
     </source>
-    <match logs>
+     # drop unused columns
+    <filter logs>
+      @type record_transformer
+      remove_keys _BOOT_ID,_CAP_EFFECTIVE,_MACHINE_ID,_SOURCE_REALTIME_TIMESTAMP,_STREAM_ID,_TRANSPORT,_SOURCE_MONOTONIC_TIMESTAMP
+    </filter>
+    # split logs to generic, audit and kubernetes runtime logs
+    <match logs >
       @type rewrite_tag_filter
       <rule>
         key MESSAGE
         pattern audit\.k8s\.io
         tag audit
       </rule>
+      <rule>
+        key CONTAINER_NAME
+        pattern ^k8s_
+        tag kubernetes
+      </rule>
       <rule>
         key MESSAGE
         pattern .+
         tag journald
       </rule>
     </match>
+    # filter kubernetes runtime logs for transformation
+    <filter kubernetes >
+      @type kubernetes_metadata
+      use_journal true
+    </filter>
+    # split kubernetes runtime logs to separate fields
+    <filter kubernetes >
+      @type record_transformer
+      enable_ruby true
+      <record>
+        POD ${record["kubernetes"]["pod_name"]}
+        NAMESPACE ${record["kubernetes"]["namespace_name"]}
+        CONTAINER_ID ${record["docker"]["container_id"]}
+        CONTAINER ${record["kubernetes"]["container_name"]}
+        HOST_MACHINE ${record["kubernetes"]["host"]}
+      </record>
+      remove_keys kubernetes,docker
+     </filter>
+     # retag splitted kubernetes logs to journald
+     <match kubernetes>
+        @type rewrite_tag_filter
+        <rule>
+          key CONTAINER
+          pattern .+
+          tag journald
+        </rule>
+     </match>
```

Comment:
If we dont retag, it goes out as new "table" and we have to modify mdsd configuration too.